modules/brainbrowser/ajax/minc.php

```diff
@@ -65,6 +65,12 @@
 }
 
 if (!empty($minc_file)) {
+    if (preg_match('/nii(\.gz)?/i', $minc_file)) {
+        header('Content-Type: application/nii');
+    } elseif (preg_match('/mnc/', $minc_file)) {
+        header('Content-Type: application/mnc');
+    }
+    header('Content-FileID: ' . $_REQUEST['minc_id']);
```

Comment:
This header name isn't standard and needs an X- prefix.

Reply:
good to know. Should be fixed now. Thanks!